src/utils.ts

```diff
@@ -28,13 +28,13 @@ export const L1_BRIDGE_ABI = new utils.Interface(require("../abi/IL1Bridge.json"
 export const L2_BRIDGE_ABI = new utils.Interface(require("../abi/IL2Bridge.json"));
 export const NONCE_HOLDER_ABI = new utils.Interface(require("../abi/INonceHolder.json"));
 
-export const ETH_ADDRESS = "0x0000000000000000000000000000000000000000";
```

Comment:
This constant is part of public API, renaming it is a breakable change. There are 2 solutions:
- keep it as it is,
- mark it as deprecated, introduce new constant `LEGACY_ETH_ADDRESS` and using it instead.

Reply:
Ok, we will have:
ETH_ADDRESS = 0x00
LEGACY_ETH_ADDRESS = 0X00
ETH_ADDRESS_IN_CONTRACTS = 0X01.